tests/rdt/transformers/test_NumberTransformer.py

```diff
@@ -79,6 +79,29 @@ def test_reverse_transform(self):
         # Check
         assert result.equals(expected_result)
 
+    def test_reverse_transform_nan(self):
+        """FIXME: finish when number and null transformer are compatible"""
```

Comment:
If there's something that is pending to be done, it's better to create an issue for it than leave a comment on the code.
Also, leave always the first line for the docstring.

Reply:
Fixed by https://github.com/HDI-Project/RDT/pull/37/commits/de3e452437d1b4cc76a6888e9f95dd4304c8e118